docs/flyte_fundamentals/registering_workflows.md

```diff
@@ -345,6 +352,15 @@ a {py:class}`~flytekit.remote.remote.FlyteRemote` object. You can learn more
 about how to do this {ref}`here <flytekit:design-control-plane>`.
 ```
 
+```{note}
+When using `pyflyte` to run jobs, the `k8sPod.dataConfig` section is automatically added to the task configuration. This section is crucial to ensure the task has access to the necessary input files.
```

Comment:
Thanks! I know this was on the issue description but given that this is a solved issue I think we can skip it altogether. It's useful but doesn't age well.

Reply:
Got it! Will remove it altogether.